librz/asm/p/asm_ppc_gnu.c

```diff
@@ -51,6 +52,14 @@ static int disassemble(RzAsm *a, RzAsmOp *op, const ut8 *buf, int len) {
 	/* prepare disassembler */
 	memset(&disasm_obj, '\0', sizeof(struct disassemble_info));
 	disasm_obj.disassembler_options = (a->bits == 64) ? "64" : "";
```

Comment:
this should not be necessary anymore

Reply:
@officialcjunior please try to remove this, and lets see if it works.